lib/bulk_update_file.rb

```diff
@@ -38,8 +38,8 @@ def bulk_update_file(dry_run:, github_token:, file_path:, file_content:, branch:
     existing_file = get_file_contents(repo_name, file_path)
     if !existing_file.nil? && file_content == Base64.decode64(existing_file.content)
       puts "⏭  file already exists with desired content"
-    elsif repo_has_branch?(repo_name, branch)
```

Comment:
Nitpick: worth noting in this (second) commit that `repo_has_branch?` will be reused in a later commit, so you've decided not to delete it here.